src/client/lobby/room-creator.js

```diff
@@ -3,10 +3,30 @@ import React, { Component } from 'react';
 import config from '../config/config';
 
 export default class RoomCreator extends Component {
+
+    onUsernameInput(){
```

Comment:
Maybe consider renaming this? To me it sounds like it would be called everytime the username input's value changes. e.g. when typing